experimental/class-brand-check-expression.md

```diff
@@ -0,0 +1,28 @@
+# [class.hasInstance](https://github.com/tc39/proposal-class-brand-check)
+
+## ClassExpression
+
+```js
+interface Class <: Node {
+    type: "Class"
+}
+```
+A class pseudo-expression.
```

Comment:
We already have https://github.com/estree/estree/blob/master/es2015.md#classes, that's why https://github.com/estree/estree/blob/master/experimental/class-access-expressions.md#class-property-access-expressions introduced `ClassObject`. I think we could share the design here.

Reply:
I agree, `ClassObject` is more suitable.